docs/computing/containers/tykky.md

```diff
@@ -107,12 +115,10 @@ Type "help", "copyright", "credits" or "license" for more information.
 >>> 
 ```
 
+### Modifying a conda installation
 
-
-## Modifying a conda installation
-
-As Tykky installed software resides in a container, it can not be directly modified.
-Small python packages can be added normally using `pip`, but then the python packages are
+Tykky installed software resides in a container, so it can not be directly modified.
+Small Python packages can be added normally using `pip`, but then the Python packages are
 sitting on the parallel filesystem so this is not recommended for any larger installations.  
 
 To actually modify the installation we can use the `update` keyword
```

Comment:
```
conda  install -y numpy
conda  remove -y pyyaml
pip install requests
```
With the new RHEL8 update, will conda installations in the  `--post-install` bash script work?

Reply:
@Nortamo ??

Reply:
Ok end-to-end example still works :)

Reply:
Errors on Mahti but not Puhti `MyEnv/bin/tput: Argument list too long` . Line 44,46,47,48,49 in `/bin/../common.sh`. But `--post-install` still works fine.